sig-contributor-experience/projects.md

```diff
@@ -0,0 +1,73 @@
+# Projects and Goals
+
+This is a list of the projects and goals currently underway with Contributor Experience. Please submit a PR if you are adding your project to this list. To introduce a new project, attend a weekly meeting or drop a note to us on the mailing list - details can be found on our [README](README.md). 
+
+Want to contribute? Take a look at this list and the "future" at the bottom. Thank you to all of our contributors for your hard work!
+
+## SIG Planning
+
+Project | Owner(s)/Lead(s) | Description | Q1, Q2, Later
```

Comment:
May be a good idea to add another column named "Status" which will have something like "Completed" when a planned project is done? This can apply to other tables in this doc as well. We could remove the item from this table as well as they are completed but keeping them here with status will make it easy for future references. Thanks!